packages/eslint-plugin/README.md

```
@@ -289,13 +420,143 @@ export default [
+   - Legacy: `"plugin:@forge-js/llm-optimized/recommended"`
+
+3. **Check plugin import:** Ensure the plugin is correctly imported and added to config
+
+4. **TypeScript issues?** Install `typescript-eslint`:
+   ```bash
+   npm install --save-dev typescript-eslint
+   ```
+
+### Common Issues
+
+**"Cannot find module '@forge-js/eslint-plugin-llm-optimized'"**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)